nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -134,36 +134,38 @@ log.info """=======================================================
 {{ cookiecutter.name }} v${workflow.manifest.version}"
 ======================================================="""
 def summary = [:]
-summary['Pipeline Name']  = '{{ cookiecutter.name }}'
+summary['Pipeline Name']    = '{{ cookiecutter.name }}'
 summary['Pipeline Version'] = workflow.manifest.version
-summary['Run Name']     = custom_runName ?: workflow.runName
+summary['Run Name']         = custom_runName ?: workflow.runName
 // TODO nf-core: Report custom parameters here
-summary['Reads']        = params.reads
-summary['Fasta Ref']    = params.fasta
-summary['Data Type']    = params.singleEnd ? 'Single-End' : 'Paired-End'
-summary['Max Memory']   = params.max_memory
-summary['Max CPUs']     = params.max_cpus
-summary['Max Time']     = params.max_time
-summary['Output dir']   = params.outdir
-summary['Working dir']  = workflow.workDir
+summary['Reads']            = params.reads
+summary['Fasta Ref']        = params.fasta
+summary['Data Type']        = params.singleEnd ? 'Single-End' : 'Paired-End'
+summary['Max Memory']       = params.max_memory
+summary['Max CPUs']         = params.max_cpus
+summary['Max Time']         = params.max_time
+summary['Output dir']       = params.outdir
+summary['Working dir']      = workflow.workDir
 summary['Container Engine'] = workflow.containerEngine
 if(workflow.containerEngine) summary['Container'] = workflow.container
-summary['Current home']   = "$HOME"
-summary['Current user']   = "$USER"
-summary['Current path']   = "$PWD"
-summary['Working dir']    = workflow.workDir
-summary['Output dir']     = params.outdir
-summary['Script dir']     = workflow.projectDir
-summary['Config Profile'] = workflow.profile
-params.config_profile_description ? summary['Config Profile Description'] = params.config_profile_description
-params.config_profile_contact ? summary['Config Profile Contact'] = params.config_profile_contact
-params.config_profile_url ? summary['Config Profile URL'] = params.config_profile_url
+summary['Current home']     = "$HOME"
+summary['Current user']     = "$USER"
+summary['Current path']     = "$PWD"
+summary['Working dir']      = workflow.workDir
+summary['Output dir']       = params.outdir
+summary['Script dir']       = workflow.projectDir
+summary['Config Profile']   = workflow.profile
+if(params.config_profile_description){
```

Comment:
```suggestion
 params.config_profile_description ?: summary['Config Profile Description'] = params.config_profile_description 
```

Reply:
Is this valid code? It doesn't look right to me..?

Reply:
```suggestion
if(params.config_profile_description){ summary['Config Profile Description'] = params.config_profile_description }
if(params.config_profile_description){ summary['Config Profile Contact'] = params.config_profile_contact }
if(params.config_profile_description){ summary['Config Profile URL'] = params.config_profile_url }
```

Reply:
They will always be defined together no? Thats why I put them under one `if` statement.

Do we need the curly braces? They havent been used in the other one-liner `if` statements in the logging. 

Im assuming the variables in the `if` statements above arent the same.

Reply:
We don't know that they will always be defined together, no. What happens if there is no obvious single URL for example? I can imagine people being unsure and omitting certain things.

Possibly don't need the curly brackets, no 👍

Reply:
And no, the other `if` blocks are different, because the contents _have_ to be defined if the statement is true.